### ISSUE TITLE
Please upgrade path-parse and css-what for security patches

### ISSUE THREAD

**Issue by stephenhmarsh** [TrySound/postcss-inline-svg#90]
**Please** merge https://github.com/TrySound/postcss-inline-svg/pull/87 and https://github.com/TrySound/postcss-inline-svg/pull/88, the GitHub UI is screaming at us.

**Comment by JenCant** [comment]
Since this has been open for so long. To fix the security warnings  for `css-what` / `css-select` that are flagged, doing the following fixes the issue and my build still works fine.

```
  // add to package.json
  "overrides": {
    "css-select": "^4.1.3"
  },
```

**Comment by Spone** [comment]
For yarn, add this to `package.json`:

```json
  "resolutions": {
    "css-select": "^4.1.3"
  },
```

**Comment by hudochenkov** [comment]
Fixed in `6.0.0`.
